src/classes/providers/proxyrack/index.ts

```diff
@@ -34,7 +34,12 @@ export class Proxyrack extends Base {
       this.setIp();
     }
 
-    return this.sendRequest(axiosRequestConfig);
+    if (this.config.strategy === EStrategyMode.CHANGE_IP_EVERY_REQUESTS) {
+      this.setIp();
+      return this.sendRequest(axiosRequestConfig);
```

Comment:
We can shorten this since `setIp()` returns  `this`, which I believe would have been the purpose
```suggestion
      return this.setIp().sendRequest(axiosRequestConfig);
```